iredis/entry.py

```diff
@@ -283,16 +283,17 @@ def print_help_msg(command):
 @click.option("-h", help="Server hostname", default="127.0.0.1")
 @click.option("-p", help="Server port", default="6379")
 @click.option("-n", help="Database number.", default=None)
+@click.option("-a", "--password", help="Server password")
```

Comment:
Let's stick with redis-cli's help

```
➜  iredis-0.1.5 redis-cli -h
redis-cli 5.0.1

Usage: redis-cli [OPTIONS] [cmd [arg [arg ...]]]
  -h <hostname>      Server hostname (default: 127.0.0.1).
  -p <port>          Server port (default: 6379).
  -s <socket>        Server socket (overrides hostname and port).
  -a <password>      Password to use when connecting to the server.
```

```suggestion
@click.option("-a", "--password", help="Password to use when connecting to the server.")
```